src/libraries/System.Console/src/System/ConsolePal.WebAssembly.cs

```diff
@@ -74,6 +76,8 @@ public override void Flush()
 
     internal static class ConsolePal
     {
+        private static readonly JSObject? s_console = (JSObject)System.Runtime.InteropServices.JavaScript.Runtime.GetGlobalObject("console");
```

Comment:
Since `Console.Clear` might not be used that much, I can move the initialization into `Clear` itself if needed.

Reply:
I think I'd prefer it happen in clear for now with a try catch that wraps the missing object exception just so we don't suddenly start throwing here if console is missing

Reply:
In my testing `System.Runtime.InteropServices.JavaScript.Runtime.GetGlobalObject` did not throw an exception if the object does not exist, and just returns `null`.

```c#
try
{
    var abc = System.Runtime.InteropServices.JavaScript.Runtime.GetGlobalObject("abc");
    if (abc is null) Console.WriteLine("abc is null");
    else Console.WriteLine($"abc is {abc.GetType().FullName}");
}
catch (Exception ex)
{
    Console.WriteLine($"{ex.GetType().FullName}: {ex.Message}\n{ex.StackTrace}");
}
````
logged
```
abc is null
```

---

Calling `Invoke("clear")` would throw if the method does not exists:
```
System.Runtime.InteropServices.JavaScript.JSException: Error: Method: 'abc' not found for: '[object Object]'
```
Should this be catched and ignored?